src/views/ManageList.jsx

```diff
@@ -1,24 +1,28 @@
 import { useCallback, useState } from 'react';
 import { useStateWithStorage } from '../utils';
-import { addItem } from '../api';
+import { addItem, shareList } from '../api';
 
 const soonDate = 7;
 const kindOfSoonDate = 14;
 const notSoonDate = 30;
 
-export function ManageList() {
+export function ManageList({ userId }) {
 	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(null);
 	const [itemName, setItemName] = useState('');
+	const [emailData, setEmailData] = useState('');
 
 	const [listPath, setListPath] = useStateWithStorage(
 		'tcl-shopping-list-path',
 		null,
 	);
 
-	// const listPath = 't4XIww03JAXm1QWr6UPEebbLRl13/first list';
-
 	const handleTextChange = (event) => {
```

Comment:
Nicely done transforming this to a unified function to handle the text change for both inputs. The `switch` approach also makes it very scalable if another input is added to the component in the future!

Reply:
I appreciate it! this was team work!